code/_onclick/hud/hud.dm

```diff
@@ -11,8 +11,11 @@ GLOBAL_LIST_INIT(available_ui_styles, list(
 	"Slimecore" = 'icons/hud/style/screen_slimecore.dmi',
 	"Operative" = 'icons/hud/style/screen_operative.dmi',
 	"Clockwork" = 'icons/hud/style/screen_clockwork.dmi',
+	"Retro" = 'icons/hud/style/screen_retro.dmi',
 	"Trasen-Knox" = 'icons/hud/style/screen_trasenknox.dmi',
-	"Retro" = 'icons/hud/style/screen_retro.dmi'
+	"Midnight-Knox" = 'icons/hud/style/screen_midnightknox.dmi',
+	"Plasma-Knox" = 'icons/hud/style/screen_plasmaknox.dmi',
+	"Nyan-knox" = 'icons/hud/style/screen_nyanknox.dmi',
```

Comment:
Replace the default midnight with this new hud, make the old midnight theme called 'legacy midnight' or something similar, remove plasmafire and slimecore completely to replace with these new designs